ACID/acid_tbc.sql

```diff
@@ -10759,87 +10759,65 @@ INSERT INTO `creature_ai_scripts` (`id`,`creature_id`,`event_type`,`event_invers
 -- =======
 -- Treant (1964) - NSR
 -- Mottled Boar (3098) - NSR
--- Dire Mottled Boar
-('309901','3099','9','0','100','1024','0','20','0','0','0','0','11','3385','0','0','0','0','0','0','0','0','0','0','Dire Mottled Boar - Cast Boar Charge'),
--- Elder Mottled Boar
-('310001','3100','9','0','100','1024','0','20','0','0','0','0','11','3385','0','0','0','0','0','0','0','0','0','0','Elder Mottled Boar - Cast Boar Charge'),
+-- Dire Mottled Boar - spell_list
```

Comment:
please keep the npc entry in the description, so one knows what list to look up.
```
Dire Mottled Boar 3099 - spell_list
```